apps/ui/src/views/SpaceUser.vue

```diff
@@ -0,0 +1,232 @@
+
+const shareMsg = computed(() => encodeURIComponent(window.location.href));
+
+const cb = computed(() => getCacheHash(user.value?.avatar));
+
+const formattedVotingPower = computed(() => {
+  const votingPower = votingPowers.value.reduce((acc, b) => acc + b.value, 0n);
+  const decimals = Math.max(...votingPowers.value.map(votingPower => votingPower.decimals), 0);
+
+  const value = _vp(Number(votingPower) / 10 ** decimals);
+
+  if (props.space.voting_power_symbol) {
+    return `${value} ${props.space.voting_power_symbol}`;
+  }
+
+  return value;
+});
+
+const navigation = computed(() => [
+  { label: 'Statement', route: 'space-user-statement' },
+  { label: 'Delegators', route: 'space-user-delegators', count: delegatesCount.value },
+  { label: 'Proposals', route: 'space-user-proposals', count: userActivity.value?.proposal_count },
+  { label: 'Latest votes', route: 'space-user-votes', count: userActivity.value?.vote_count }
+]);
+
+async function loadUserActivity() {
+  const spaceNetwork = getNetwork(props.space.network);
+
+  const users = await spaceNetwork.api.loadLeaderboard(
+    props.space.id,
+    {
+      limit: 1,
+      skip: 0
+    },
+    'vote_count-desc',
+    userId.value
+  );
+
+  if (users[0]) userActivity.value = users[0];
+}
+
+async function loadDelegates() {
+  delegatesCount.value = 0;
+
+  const delegates = (
+    await Promise.all(
+      props.space.delegations.map(async delegation => {
+        const { getDelegates } = useDelegates(
+          delegation.apiUrl as string,
+          delegation.contractAddress as string
+        );
+
+        return (
+          await getDelegates({
+            orderBy: 'delegatedVotes',
+            orderDirection: 'asc',
+            skip: 0,
+            first: 1,
+            user: userId.value
+          })
+        )[0];
+      })
+    )
+  )
+    .flat()
+    .filter(Boolean);
+
+  delegatesCount.value = delegates
+    .map(delegate => delegate.tokenHoldersRepresentedAmount || 0)
+    .reduce((a, b) => a + b, 0);
+}
+
+async function getVotingPower() {
+  votingPowerStatus.value = 'loading';
+  try {
+    votingPowers.value = await network.value.actions.getVotingPower(
+      props.space.id,
+      props.space.strategies,
+      props.space.strategies_params,
+      props.space.strategies_parsed_metadata,
+      userId.value,
+      {
+        at: supportsNullCurrent(props.space.network) ? null : getCurrent(props.space.network) || 0,
+        chainId: props.space.snapshot_chain_id
+      }
+    );
+    votingPowerStatus.value = 'success';
+  } catch (e) {
+    console.warn('Failed to load voting power', e);
+    votingPowers.value = [];
+    votingPowerStatus.value = 'error';
+  }
+}
+
+watch(
+  userId,
+  async id => {
+    loaded.value = false;
+
+    if (isValidAddress(id)) {
+      await usersStore.fetchUser(id);
+      await loadUserActivity();
+      loadDelegates();
+      getVotingPower();
+    }
+
+    loaded.value = true;
+  },
+  { immediate: true }
```

Comment:
If i go here http://localhost:8080/#/sn:0x039a5539bf1866da915779bb6270d1b839ca82cb0acd4eefbc17ba99ebc3a021/profile/0x013477227a72edc53a237231ec9f4c7180c93386664b1e3a1b5d181c354e5f39

Replace user address with `0x03c4dd9305d6b3a2c1ba7ed06c2285125751600c821ec45279e3760db341d67e` in URL, who doesn't any votes or proposals on this proposal
It shows previous values 🙈 when i reload it works fine